lib/core/src/display/render/passes/screen.rs

```diff
@@ -23,12 +23,15 @@ impl ScreenRenderPass {
     /// Constructor.
     pub fn new(world:&World) -> Self {
         let screen = Screen::new(world);
+        screen.hide();
         Self {screen}
     }
 }
 
 impl RenderPass for ScreenRenderPass {
     fn run(&mut self, _:&Context, _:&UniformScope) {
+        self.screen.show();
```

Comment:
I don't get it. Why we hide the show screen for rendering and then hide? A comment would be helpful.

Reply:
Added docs.